scripts/generate-charts

```diff
@@ -15,18 +15,26 @@ for f in packages/*; do
 			yq w -i ${f}/charts/Chart.yaml 'version' "${version}${packageVersion}"
 		fi
 		helm package ${f}/charts --destination assets/$(basename -- ${f})
-		git checkout assets/$(basename -- ${f})/"$(basename -- ${f})-$(yq r ${f}/charts/Chart.yaml version).tgz" 2>/dev/null || true
 		if [[ -d ${f}/charts-crd ]]; then
 			packageVersion=$(yq r ${f}/package.yaml packageVersion)
 			yq w -i ${f}/charts-crd/Chart.yaml 'version' "${version}${packageVersion}"
 
 			helm package ${f}/charts-crd --destination assets/$(basename -- ${f})
-			git checkout assets/$(basename -- ${f})/"$(basename -- ${f})-crd-$(yq r ${f}/charts-crd/Chart.yaml version).tgz" || true
 		fi
+		helm repo index --merge ./assets/index.yaml --url assets assets
```

Comment:
We no longer checkout this in order to prevent user overriding existing chart. This can't done in the new scripts. Developer has to be very carefully in overrding charts as any changed introduced in the chart will get written into the current version of helm tarball.

Reply:
Not following this, we can chat offline.